.github/workflows/main.yml

```diff
@@ -12,9 +12,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 14
     - run: npm install
     - run: npx aegir lint
-    - uses: gozala/typescript-error-reporter-action@v1.0.8
```

Comment:
This was failing with:

```
Run gozala/typescript-error-reporter-action@v1.0.8
Using local typescript@4.3.5
Error: Could not find a declaration file for module 'multibase'. '/home/runner/work/js-multihash/js-multihash/node_modules/multibase/src/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/multibase` if it exists or add a new declaration (.d.ts) file containing `declare module 'multibase';`
Error: Found 1 errors!
```
multibase does not have types, but even with an ignore it fails.